red-hat-data-services/odh-manifests
# Add JSP gpu toleration for 'nvidia.com/gpu' EXISTS

Add a Jupyter singleuser profile gpuTypes for the toleration nvidia.com/gpu EXISTS.

Test steps

1. Install RHODS and wait for RHODS install to complete
1. Add a gpu machine pool with a NoSchedule node taint of `nvidia.com/gpu = true` (Value is required by OCM but NVIDIA only checks for existence and this JH profile only check for existence) --AND-- GPU instances that are supported by the NVIDIA gpu addon
1. Deploy the NVIDIA gpu addon and wait for it to finish installing
1. Spawn a PyTorch or TensorFlow notebook with 1 or more gpus requested
1. Confirm that the notebook pod spec has a toleration for `nvidia.com/gpu EXISTS` and it is placed on a gpu node

Jira: RHODS-540

Signed-off-by: Landon LaSmith <LLaSmith@redhat.com>

- [x] JIRA link(s): https://issues.redhat.com/browse/RHODS-540
